docker-compose.yml

```diff
@@ -12,7 +12,8 @@ services:
     volumes:
       - pgdata:/var/lib/postgresql/data
       - ./docker-entrypoint.postgres.sql:/docker-entrypoint-initdb.d/init.sql
-      - ./docker-postgresql.conf:/var/lib/postgresql/data/postgresql.conf
+      - ./docker-postgresql.conf:/etc/postgresql/postgresql.conf
+    command: postgres -c config_file=/etc/postgresql/postgresql.conf
```

Comment:
Hmm, this shouldn't be needed since it works locally in the default location 🤔

Reply:
If that is not in place, the CI database fails on init. Error message says that the `/var/lib/postgresql/data` location is not empty.  Not really sure why it works fine locally.